google/cloud/aiplatform/vizier/study.py

```diff
@@ -232,22 +232,34 @@ def materialize_study_config(self) -> vz.StudyConfig:
         study = self.api_client.get_study(
             name=self.resource_name, credentials=self.credentials
         )
-        return copy.deepcopy(vz.StudyConfig.from_proto(study.study_spec))
+        return copy.deepcopy(
+            vz.ProblemStatement.from_problem(
+                vz.StudyConfig.from_proto(study.study_spec)
+            )
+        )
 
     @classmethod
-    def from_uid(
+    def from_resource_name(
         cls: Type[_T],
-        uid: str,
+        name: str,
```

Comment:
same comment as above. This is a breaking change for customers who use keyword arguments in their function calls